package.json

```diff
@@ -142,5 +142,10 @@
     "testMatch": [
       "**/*.test.js"
     ]
+  },
+  "prettier": {
```

Comment:
@thymikee could you test if this makes any of the config here unnecessary?

https://github.com/facebook/jest/blob/8901f8752fe6a1369b94ea06edfaca48de3f7c8c/.vscode/settings.json#L12-L17

Reply:
It was cleaner to put it here instead of inline in the run script

Reply:
We should be safe to remove all of these except for `prettier.eslintIntegration`, because `eslint --fix` is responsible for formatting.

Reply:
Read the prettier-vscode docs, and the config in there is actually ignored if there's a `prettier` key in package.json.

https://github.com/prettier/prettier-vscode/blob/221e65c8e245aaea02dc4deb2a566680c3f73caa/README.md#prettiers-settings